### PR TITLE
Integrate react implementation of icon only client interface

### PR DESCRIPTION
**Pull request by kldavis4**

### CODE REVIEW

**Comment by kldavis4** [comment]
@edwinsteele have a look at this when you get a chance. Does this integration with the main project look ok? Also, if you have time, feel free to try out the new react implementation and let me know if you see any issues. I've just finished up implementing a good number of tests as well as the system file filtering feature.

File: ansible/roles/webserver-content/tasks/main.yml [hunk 1]
```diff
@@ -27,12 +27,26 @@
     - "{{ connectbox_default_content_root }}"
     - "{{ connectbox_captive_portal_root }}"
 
-# Copy client interface files
-- name: Copy connectbox client interface
+# Copy local client interface files
+- name: Copy local connectbox client interface
+  when: connectbox_client_repo == ""
```
Comment (edwinsteele) [inline ansible/roles/webserver-content/tasks/main.yml]: It’s not clear why these `when:` statements exist. I can imagine specifying an alternate repo URL would help with development, but it’s unclear why the repo would ever be undefined. If they need to be here, please add a brief comment on why the `when:` statements exist.

Comment (kldavis4) [inline ansible/roles/webserver-content/tasks/main.yml]: Thanks for the review.

So, my reasoning was we would leave the original angular icon interface as a default if the connectbox_client_repo was left blank. If you don't think there is a good case for that, I'm fine with ripping it out and removing the when statements.

Comment (edwinsteele) [inline ansible/roles/webserver-content/tasks/main.yml]: Let's rip out the old one and embrace the future :-D

**Pull request merged**